src/Pages/Event/Event.js

```diff
@@ -2,49 +2,58 @@ import React, { useEffect } from 'react'
+    //     ,topImage:"/images/bg1.png",color:"#ffd34e"}
+    // ])
 
-    useEffect(() => {
-        setChosen(chosen)
-    }, [chosen])
+    // const[upcomingdata,setUpcomingData]=useState([
+    //     {id:"1",date:"11",month:"jan 23",topic:"Orientation Session",text:"We are having a live session where we will introduce you to Google Developer Students ClubsWe are having a live session where we will introduce you to Google Developer Students ClubsWe are having a live session where we will introduce you to Google Developer Students Clubs"
+    //     ,topImage:"/images/bg1.png",color:"#ffd34e"},
+    //     {id:"2",date:"12",month:"jan 23",topic:"Orientation Session",text:"We are having a live session where we will introduce you to Google Developer Students ClubsWe are having a live session where we will introduce you to Google Developer Students ClubsWe are having a live session where we will introduce you to Google Developer Students Clubs"
+    //     ,topImage:"/images/bg2.png",color:"#f27770"}
+    //     ,{id:"3",date:"03",month:"jan 23",topic:"Orientation Session",text:"We are having a live session where we will introduce you to Google Developer Students ClubsWe are having a live session where we will introduce you to Google Developer Students ClubsWe are having a live session where we will introduce you to Google Developer Students Clubs"
+    //     ,topImage:"/images/bg3.png",color:"#3eaff4"},
+    //     {id:"4",date:"04",month:"jan 23",topic:"Orientation Session",text:"We are having a live session where we will introduce you to Google Developer Students ClubsWe are having a live session where we will introduce you to Google Developer Students ClubsWe are having a live session where we will introduce you to Google Developer Students Clubs"
+    //     ,topImage:"/images/bg4.png",color:"#6ebb71"},
+    //     {id:"5",date:"06",month:"jan 23",topic:"Orientation Session",text:"We are having a live session where we will introduce you to Google Developer Students ClubsWe are having a live session where we will introduce you to Google Developer Students ClubsWe are having a live session where we will introduce you to Google Developer Students Clubs"
+    //     ,topImage:"/images/bg2.png",color:"#f47169"},
+    //     {id:"6",date:"06",month:"jan 23",topic:"Orientation Session",text:"We are having a live session where we will introduce you to Google Developer Students ClubsWe are having a live session where we will introduce you to Google Developer Students ClubsWe are having a live session where we will introduce you to Google Developer Students Clubs"
+    //     ,topImage:"/images/bg1.png",color:"#ffd34e"}
+    // ])
+    // const allEvents = upcomingdata.concat(pastData) 
+    // /* allEvents is an array which consists of both upcoming and past events */
+    // const [allData, setAllData] = useState(allEvents)
+
+    // useEffect(() => {
+    //     setAllData(allData)
+    // }, [allData])
+
+    // useEffect(() => {
```

Comment:
@aryanda1 if we are not using these line of code can we delete them or there is any specific reason for keeping them

Reply:
Well they were in the original code. And i saw in some other PR code, they commented out the old stuff. So i wasnt sure what to do. Should i delete it?

Reply:
Then please delete them and then I think we will be good to go 😎

Reply:
Done check